FreeRTOS-Plus/Demo/AWS/Mqtt_Demo_Helpers/mqtt_pkcs11_demo_helpers.c

```diff
@@ -111,8 +111,11 @@
 
 /**
  * @brief Timeout for MQTT_ProcessLoop in milliseconds.
+ * This timeout corresponds to time taken by Number of topic subscribed to publish
+ * messages from device to broker and broker to device to complete.
+ * Any change in Number of topic subscribed needs updating mqttexamplePROCESS_LOOP_TIMEOUT_MS.
```

Comment:
Please check once if we also have to update the mqttexampleOUTGOING_PUBLISH_RECORD_LEN in mqtt_pkcs11_demo_helpers.c and mqtt_demo_helpers.c